File: demos/retrofit-demo-java/app/src/main/java/com/smd/retrofitdemo/network/Person.java
package com.smd.retrofitdemo.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * A person in space, as defined by the API response
 */
public class Person {

    @SerializedName("craft")
    @Expose
    private String craft;
    @SerializedName("name")
    @Expose
    private String name;

    public String getCraft() {
        return craft;
    }

    public void setCraft(String craft) {
        this.craft = craft;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Person{" +
                "craft='" + craft + '\'' +
                ", name='" + name + '\'' +
                '}';
    }
}

File: lab5-demos/retrofit-demo-kotlin/app/src/main/java/com/smd/retrofitdemo/network/Person.kt
package com.smd.retrofitdemo.network

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * A person in space, as defined by the API response
 */
@JsonClass(generateAdapter = true)
data class Person(
    @Json(name = "craft")
    var craft: String,

    @Json(name = "name")
    var name: String
)